commands/cosmetics.py

```diff
@@ -15,9 +15,9 @@ def __init__(self, client):
     async def cosmetics(self, ctx, operand = None, cosmetic = None):
         if operand is None:
             #get inventory
-            cosmetics_all = sql.db_query("ibm.db", "SELECT cosmetic_id FROM cosmetics")
+            cosmetics_all = sql.db_query("SELECT cosmetic_id FROM cosmetics")
             cosmetics_total = len(cosmetics_all)
-            inventory = eval(sql.db_query("ibm.db", "SELECT cosmetics FROM Members WHERE UserID = %s" % (str(ctx.author.id)))[0][0])
+            inventory = eval(sql.db_query("SELECT cosmetics FROM Members WHERE UserID = %s" % (str(ctx.author.id)))[0][0])

```

Comment:
Maybe use sql COUNT instead of eval here

Reply:
This is out of scope for this PR. Please raise an issue instead.